src/Elmish.WPF/ViewModel.fs

```diff
@@ -238,39 +231,37 @@ and [<AllowNullLiteral>] internal ViewModel<'model, 'msg>
             let preventClose = ref true
             log "[%s] Creating hidden window" chain
             showNewWindow
-              winRef d.GetWindow vm d.IsModal d.OnCloseRequested
-              preventClose Visibility.Hidden dispatch
+              winRef d.GetWindow vm d.IsModal onCloseRequested
+              preventClose Visibility.Hidden
             SubModelWin (
               ref <| WindowState.Hidden vm,
               getState,
               getBindings,
               toMsg,
               d.GetWindow,
               d.IsModal,
-              d.OnCloseRequested,
+              onCloseRequested,
               preventClose,
-              winRef
-            )
+              winRef)
```

Comment:
For symmetry I generally like to keep the closing parenthesis at the same level of indentation as the line of the start parenthesis, so I'd like to keep this as it is.

Reply:
Ok.  I had changed these two to make them consistent with `WeakReference<_>(null))` above.  I just changed these two back and also changed the first case to match.

P.S.
When I don't make the exact change you request (like I did here), I will leave the conversation unresolved so I don't sneak anything by you ;)

Reply:
Thanks!